opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -239,16 +226,16 @@ message Span {
   }
 
   // The included links.
-  Links links = 13;
+  Links links = 14;
 
   // An optional final status for this span. Semantically when Status
   // wasn't set it is means span ended without errors and assume
   // Status.Ok (code = 0).
-  Status status = 14;
+  Status status = 15;
 
   // An optional number of child spans that were generated while this span
   // was active. If set, allows an implementation to detect missing child spans.
-  google.protobuf.UInt32Value child_span_count = 15;
+  google.protobuf.UInt32Value child_span_count = 16;
```

Comment:
Update the next index at the top of the file?

Reply:
Done.